[2021-09-02 13:09:09] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:09:09] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:09:09] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:09:09] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:09:09] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:09:09] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:09:09] [DEBUG] The application and model name are [('api', 'apiproject')]
[2021-09-02 13:09:09] [DEBUG] The application and model name are [('api', 'templates')]
[2021-09-02 13:09:09] [DEBUG] The application and model name are [('api', 'testsuit')]
[2021-09-02 13:09:09] [DEBUG] The application and model name are [('api', 'testcase')]
[2021-09-02 13:09:09] [DEBUG] The application and model name are [('api', 'scenario')]
[2021-09-02 13:09:09] [DEBUG] The application and model name are [('api', 'sql')]
[2021-09-02 13:09:09] [DEBUG] Success to import application :[api]
[2021-09-02 13:09:09] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:09:09] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:09:09] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.public_admin'
[2021-09-02 13:09:09] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.public_admin'
[2021-09-02 13:09:09] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.public_admin'
[2021-09-02 13:09:09] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.public_admin'
[2021-09-02 13:09:09] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.public_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.public_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [api],due to error: No module named 'api.public_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [core],due to error: No module named 'core.public_admin'
[2021-09-02 13:09:10] [DEBUG] The application and model name are [('public', 'userprofile')]
[2021-09-02 13:09:10] [DEBUG] The application and model name are [('public', 'groups')]
[2021-09-02 13:09:10] [DEBUG] The application and model name are [('public', 'role')]
[2021-09-02 13:09:10] [DEBUG] The application and model name are [('public', 'firstlayermenu')]
[2021-09-02 13:09:10] [DEBUG] The application and model name are [('public', 'submenu')]
[2021-09-02 13:09:10] [DEBUG] The application and model name are [('public', 'businessfunc')]
[2021-09-02 13:09:10] [DEBUG] The application and model name are [('public', 'retrieval')]
[2021-09-02 13:09:10] [DEBUG] The application and model name are [('public', 'resource')]
[2021-09-02 13:09:10] [DEBUG] The application and model name are [('public', 'environment')]
[2021-09-02 13:09:10] [DEBUG] The application and model name are [('public', 'dbinfo')]
[2021-09-02 13:09:10] [DEBUG] The application and model name are [('public', 'logserverinfo')]
[2021-09-02 13:09:10] [DEBUG] The application and model name are [('public', 'configurations')]
[2021-09-02 13:09:10] [DEBUG] The application and model name are [('public', 'cn_en_map')]
[2021-09-02 13:09:10] [DEBUG] The application and model name are [('public', 'steplog')]
[2021-09-02 13:09:10] [DEBUG] The application and model name are [('public', 'todolist')]
[2021-09-02 13:09:10] [DEBUG] Success to import application :[public]
[2021-09-02 13:09:10] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:09:10] [DEBUG] Success to import application :[api]
[2021-09-02 13:09:10] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.web_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.web_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.web_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.web_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.web_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.web_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [api],due to error: No module named 'api.web_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [core],due to error: No module named 'core.web_admin'
[2021-09-02 13:09:10] [ERROR] Fail to import application: [public],due to error: No module named 'public.web_admin'
[2021-09-02 13:09:13] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:09:13] [INFO] The user is Administrator!
[2021-09-02 13:10:12] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:10:12] [INFO] The user is Administrator!
[2021-09-02 13:10:12] [DEBUG] There is a file [<MultiValueDict: {'photo': [<InMemoryUploadedFile: 微信图片_20210618153322.jpg (image/jpeg)>]}>] to be uploaded!
[2021-09-02 13:10:12] [ERROR] Fail to upload the file,error as follows: 'upload'
[2021-09-02 13:18:20] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:18:20] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:18:20] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:18:20] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:18:20] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:18:20] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:18:20] [DEBUG] The application and model name are [('api', 'apiproject')]
[2021-09-02 13:18:20] [DEBUG] The application and model name are [('api', 'templates')]
[2021-09-02 13:18:20] [DEBUG] The application and model name are [('api', 'testsuit')]
[2021-09-02 13:18:20] [DEBUG] The application and model name are [('api', 'testcase')]
[2021-09-02 13:18:20] [DEBUG] The application and model name are [('api', 'scenario')]
[2021-09-02 13:18:20] [DEBUG] The application and model name are [('api', 'sql')]
[2021-09-02 13:18:20] [DEBUG] Success to import application :[api]
[2021-09-02 13:18:20] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:18:20] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.public_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.public_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.public_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.public_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.public_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.public_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [api],due to error: No module named 'api.public_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [core],due to error: No module named 'core.public_admin'
[2021-09-02 13:18:21] [DEBUG] The application and model name are [('public', 'userprofile')]
[2021-09-02 13:18:21] [DEBUG] The application and model name are [('public', 'groups')]
[2021-09-02 13:18:21] [DEBUG] The application and model name are [('public', 'role')]
[2021-09-02 13:18:21] [DEBUG] The application and model name are [('public', 'firstlayermenu')]
[2021-09-02 13:18:21] [DEBUG] The application and model name are [('public', 'submenu')]
[2021-09-02 13:18:21] [DEBUG] The application and model name are [('public', 'businessfunc')]
[2021-09-02 13:18:21] [DEBUG] The application and model name are [('public', 'retrieval')]
[2021-09-02 13:18:21] [DEBUG] The application and model name are [('public', 'resource')]
[2021-09-02 13:18:21] [DEBUG] The application and model name are [('public', 'environment')]
[2021-09-02 13:18:21] [DEBUG] The application and model name are [('public', 'dbinfo')]
[2021-09-02 13:18:21] [DEBUG] The application and model name are [('public', 'logserverinfo')]
[2021-09-02 13:18:21] [DEBUG] The application and model name are [('public', 'configurations')]
[2021-09-02 13:18:21] [DEBUG] The application and model name are [('public', 'cn_en_map')]
[2021-09-02 13:18:21] [DEBUG] The application and model name are [('public', 'steplog')]
[2021-09-02 13:18:21] [DEBUG] The application and model name are [('public', 'todolist')]
[2021-09-02 13:18:21] [DEBUG] Success to import application :[public]
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:18:21] [DEBUG] Success to import application :[api]
[2021-09-02 13:18:21] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.web_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.web_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.web_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.web_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.web_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.web_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [api],due to error: No module named 'api.web_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [core],due to error: No module named 'core.web_admin'
[2021-09-02 13:18:21] [ERROR] Fail to import application: [public],due to error: No module named 'public.web_admin'
[2021-09-02 13:18:29] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:18:29] [INFO] The user is Administrator!
[2021-09-02 13:18:33] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:18:33] [INFO] The user is Administrator!
[2021-09-02 13:18:33] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 13:18:33] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 13:18:33] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 13:18:33] [DEBUG] The verbose name of this many to many field is: 角色
[2021-09-02 13:18:33] [DEBUG] The verbose name of this many to many field is: 权限组
[2021-09-02 13:18:33] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:18:33] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:18:33] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:18:35] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:18:35] [INFO] The user is Administrator!
[2021-09-02 13:18:45] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:18:45] [INFO] The user is Administrator!
[2021-09-02 13:18:45] [DEBUG] There is a file [<MultiValueDict: {'photo': [<InMemoryUploadedFile: diaochan.jpg (image/jpeg)>]}>] to be uploaded,it is a update action!
[2021-09-02 13:18:47] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:18:47] [INFO] The user is Administrator!
[2021-09-02 13:18:47] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 13:18:47] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 13:18:47] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 13:18:47] [DEBUG] The verbose name of this many to many field is: 角色
[2021-09-02 13:18:47] [DEBUG] The verbose name of this many to many field is: 权限组
[2021-09-02 13:18:47] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:18:47] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:18:47] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:19:19] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:19:19] [INFO] The user is Administrator!
[2021-09-02 13:19:22] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:19:22] [INFO] The user is Administrator!
[2021-09-02 13:19:22] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:19:22] [INFO] The user is Administrator!
[2021-09-02 13:19:22] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 13:19:22] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 13:19:22] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 13:19:22] [DEBUG] The verbose name of this many to many field is: 角色
[2021-09-02 13:19:22] [DEBUG] The verbose name of this many to many field is: 权限组
[2021-09-02 13:19:22] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:19:22] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:19:22] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:19:25] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:19:25] [INFO] The user is Administrator!
[2021-09-02 13:19:38] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:19:38] [INFO] The user is Administrator!
[2021-09-02 13:19:38] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 13:19:38] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 13:19:38] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 13:19:38] [DEBUG] The verbose name of this many to many field is: 角色
[2021-09-02 13:19:38] [DEBUG] The verbose name of this many to many field is: 权限组
[2021-09-02 13:19:38] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:19:38] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:19:38] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:19:40] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:19:40] [INFO] The user is Administrator!
[2021-09-02 13:19:48] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:19:48] [INFO] The user is Administrator!
[2021-09-02 13:19:48] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 13:19:48] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 13:19:48] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 13:19:48] [DEBUG] The verbose name of this many to many field is: 角色
[2021-09-02 13:19:48] [DEBUG] The verbose name of this many to many field is: 权限组
[2021-09-02 13:19:48] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:19:48] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:19:48] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('api', 'apiproject')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('api', 'templates')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('api', 'testsuit')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('api', 'testcase')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('api', 'scenario')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('api', 'sql')]
[2021-09-02 13:23:39] [DEBUG] Success to import application :[api]
[2021-09-02 13:23:39] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.public_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.public_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.public_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.public_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.public_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.public_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [api],due to error: No module named 'api.public_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [core],due to error: No module named 'core.public_admin'
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('public', 'userprofile')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('public', 'groups')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('public', 'role')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('public', 'firstlayermenu')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('public', 'submenu')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('public', 'businessfunc')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('public', 'retrieval')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('public', 'resource')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('public', 'environment')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('public', 'dbinfo')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('public', 'logserverinfo')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('public', 'configurations')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('public', 'cn_en_map')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('public', 'steplog')]
[2021-09-02 13:23:39] [DEBUG] The application and model name are [('public', 'todolist')]
[2021-09-02 13:23:39] [DEBUG] Success to import application :[public]
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:23:39] [DEBUG] Success to import application :[api]
[2021-09-02 13:23:39] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.web_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.web_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.web_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.web_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.web_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.web_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [api],due to error: No module named 'api.web_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [core],due to error: No module named 'core.web_admin'
[2021-09-02 13:23:39] [ERROR] Fail to import application: [public],due to error: No module named 'public.web_admin'
[2021-09-02 13:23:40] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:23:40] [INFO] The user is Administrator!
[2021-09-02 13:23:40] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 13:23:40] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 13:23:40] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 13:23:40] [DEBUG] The verbose name of this many to many field is: 角色
[2021-09-02 13:23:40] [DEBUG] The verbose name of this many to many field is: 权限组
[2021-09-02 13:23:40] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:23:40] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:23:40] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:24:16] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:24:16] [INFO] The user is Administrator!
[2021-09-02 13:24:16] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 13:24:16] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 13:24:16] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 13:24:16] [DEBUG] The verbose name of this many to many field is: 角色
[2021-09-02 13:24:16] [DEBUG] The verbose name of this many to many field is: 权限组
[2021-09-02 13:24:16] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:24:16] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:24:16] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:24:18] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:24:18] [INFO] The user is Administrator!
[2021-09-02 13:24:46] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:24:46] [INFO] The user is Administrator!
[2021-09-02 13:24:46] [DEBUG] There is a file [<MultiValueDict: {'photo': [<InMemoryUploadedFile: baishe.jpg (image/jpeg)>]}>] to be uploaded!
[2021-09-02 13:24:46] [ERROR] Fail to upload the file,error as follows: 'upload'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('api', 'apiproject')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('api', 'templates')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('api', 'testsuit')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('api', 'testcase')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('api', 'scenario')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('api', 'sql')]
[2021-09-02 13:26:07] [DEBUG] Success to import application :[api]
[2021-09-02 13:26:07] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.public_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.public_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.public_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.public_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.public_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.public_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [api],due to error: No module named 'api.public_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [core],due to error: No module named 'core.public_admin'
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('public', 'userprofile')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('public', 'groups')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('public', 'role')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('public', 'firstlayermenu')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('public', 'submenu')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('public', 'businessfunc')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('public', 'retrieval')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('public', 'resource')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('public', 'environment')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('public', 'dbinfo')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('public', 'logserverinfo')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('public', 'configurations')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('public', 'cn_en_map')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('public', 'steplog')]
[2021-09-02 13:26:07] [DEBUG] The application and model name are [('public', 'todolist')]
[2021-09-02 13:26:07] [DEBUG] Success to import application :[public]
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:26:07] [DEBUG] Success to import application :[api]
[2021-09-02 13:26:07] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.web_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.web_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.web_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.web_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.web_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.web_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [api],due to error: No module named 'api.web_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [core],due to error: No module named 'core.web_admin'
[2021-09-02 13:26:07] [ERROR] Fail to import application: [public],due to error: No module named 'public.web_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('api', 'apiproject')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('api', 'templates')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('api', 'testsuit')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('api', 'testcase')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('api', 'scenario')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('api', 'sql')]
[2021-09-02 13:26:16] [DEBUG] Success to import application :[api]
[2021-09-02 13:26:16] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.public_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.public_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.public_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.public_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.public_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.public_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [api],due to error: No module named 'api.public_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [core],due to error: No module named 'core.public_admin'
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('public', 'userprofile')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('public', 'groups')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('public', 'role')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('public', 'firstlayermenu')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('public', 'submenu')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('public', 'businessfunc')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('public', 'retrieval')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('public', 'resource')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('public', 'environment')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('public', 'dbinfo')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('public', 'logserverinfo')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('public', 'configurations')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('public', 'cn_en_map')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('public', 'steplog')]
[2021-09-02 13:26:16] [DEBUG] The application and model name are [('public', 'todolist')]
[2021-09-02 13:26:16] [DEBUG] Success to import application :[public]
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:26:16] [DEBUG] Success to import application :[api]
[2021-09-02 13:26:16] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.web_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.web_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.web_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.web_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.web_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.web_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [api],due to error: No module named 'api.web_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [core],due to error: No module named 'core.web_admin'
[2021-09-02 13:26:16] [ERROR] Fail to import application: [public],due to error: No module named 'public.web_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('api', 'apiproject')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('api', 'templates')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('api', 'testsuit')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('api', 'testcase')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('api', 'scenario')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('api', 'sql')]
[2021-09-02 13:26:28] [DEBUG] Success to import application :[api]
[2021-09-02 13:26:28] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.public_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.public_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.public_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.public_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.public_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.public_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [api],due to error: No module named 'api.public_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [core],due to error: No module named 'core.public_admin'
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('public', 'userprofile')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('public', 'groups')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('public', 'role')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('public', 'firstlayermenu')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('public', 'submenu')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('public', 'businessfunc')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('public', 'retrieval')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('public', 'resource')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('public', 'environment')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('public', 'dbinfo')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('public', 'logserverinfo')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('public', 'configurations')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('public', 'cn_en_map')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('public', 'steplog')]
[2021-09-02 13:26:28] [DEBUG] The application and model name are [('public', 'todolist')]
[2021-09-02 13:26:28] [DEBUG] Success to import application :[public]
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:26:28] [DEBUG] Success to import application :[api]
[2021-09-02 13:26:28] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.web_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.web_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.web_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.web_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.web_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.web_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [api],due to error: No module named 'api.web_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [core],due to error: No module named 'core.web_admin'
[2021-09-02 13:26:28] [ERROR] Fail to import application: [public],due to error: No module named 'public.web_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('api', 'apiproject')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('api', 'templates')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('api', 'testsuit')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('api', 'testcase')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('api', 'scenario')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('api', 'sql')]
[2021-09-02 13:26:46] [DEBUG] Success to import application :[api]
[2021-09-02 13:26:46] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.public_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.public_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.public_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.public_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.public_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.public_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [api],due to error: No module named 'api.public_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [core],due to error: No module named 'core.public_admin'
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('public', 'userprofile')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('public', 'groups')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('public', 'role')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('public', 'firstlayermenu')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('public', 'submenu')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('public', 'businessfunc')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('public', 'retrieval')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('public', 'resource')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('public', 'environment')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('public', 'dbinfo')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('public', 'logserverinfo')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('public', 'configurations')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('public', 'cn_en_map')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('public', 'steplog')]
[2021-09-02 13:26:46] [DEBUG] The application and model name are [('public', 'todolist')]
[2021-09-02 13:26:46] [DEBUG] Success to import application :[public]
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:26:46] [DEBUG] Success to import application :[api]
[2021-09-02 13:26:46] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.web_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.web_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.web_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.web_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.web_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.web_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [api],due to error: No module named 'api.web_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [core],due to error: No module named 'core.web_admin'
[2021-09-02 13:26:46] [ERROR] Fail to import application: [public],due to error: No module named 'public.web_admin'
[2021-09-02 13:26:50] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:26:50] [INFO] The user is Administrator!
[2021-09-02 13:26:57] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:26:57] [INFO] The user is Administrator!
[2021-09-02 13:26:57] [DEBUG] There is a file [<MultiValueDict: {'photo': [<InMemoryUploadedFile: baishe.jpg (image/jpeg)>]}>] to be uploaded!
[2021-09-02 13:26:57] [ERROR] Fail to upload the file,error as follows: 'NoneType' object has no attribute 'name'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('api', 'apiproject')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('api', 'templates')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('api', 'testsuit')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('api', 'testcase')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('api', 'scenario')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('api', 'sql')]
[2021-09-02 13:29:17] [DEBUG] Success to import application :[api]
[2021-09-02 13:29:17] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.public_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.public_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.public_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.public_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.public_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.public_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [api],due to error: No module named 'api.public_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [core],due to error: No module named 'core.public_admin'
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('public', 'userprofile')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('public', 'groups')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('public', 'role')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('public', 'firstlayermenu')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('public', 'submenu')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('public', 'businessfunc')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('public', 'retrieval')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('public', 'resource')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('public', 'environment')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('public', 'dbinfo')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('public', 'logserverinfo')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('public', 'configurations')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('public', 'cn_en_map')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('public', 'steplog')]
[2021-09-02 13:29:17] [DEBUG] The application and model name are [('public', 'todolist')]
[2021-09-02 13:29:17] [DEBUG] Success to import application :[public]
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:29:17] [DEBUG] Success to import application :[api]
[2021-09-02 13:29:17] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.web_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.web_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.web_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.web_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.web_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.web_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [api],due to error: No module named 'api.web_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [core],due to error: No module named 'core.web_admin'
[2021-09-02 13:29:17] [ERROR] Fail to import application: [public],due to error: No module named 'public.web_admin'
[2021-09-02 13:29:18] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:29:18] [INFO] The user is Administrator!
[2021-09-02 13:29:18] [DEBUG] There is a file [<MultiValueDict: {'photo': [<InMemoryUploadedFile: baishe.jpg (image/jpeg)>]}>] to be uploaded!
[2021-09-02 13:29:18] [ERROR] Fail to upload the file,error as follows: 'NoneType' object has no attribute 'name'
[2021-09-02 13:29:42] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:29:42] [INFO] The user is Administrator!
[2021-09-02 13:29:46] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:29:46] [INFO] The user is Administrator!
[2021-09-02 13:29:46] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 13:29:46] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 13:29:46] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('api', 'apiproject')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('api', 'templates')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('api', 'testsuit')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('api', 'testcase')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('api', 'scenario')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('api', 'sql')]
[2021-09-02 13:30:15] [DEBUG] Success to import application :[api]
[2021-09-02 13:30:15] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.public_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.public_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.public_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.public_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.public_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.public_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [api],due to error: No module named 'api.public_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [core],due to error: No module named 'core.public_admin'
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('public', 'userprofile')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('public', 'groups')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('public', 'role')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('public', 'firstlayermenu')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('public', 'submenu')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('public', 'businessfunc')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('public', 'retrieval')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('public', 'resource')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('public', 'environment')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('public', 'dbinfo')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('public', 'logserverinfo')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('public', 'configurations')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('public', 'cn_en_map')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('public', 'steplog')]
[2021-09-02 13:30:15] [DEBUG] The application and model name are [('public', 'todolist')]
[2021-09-02 13:30:15] [DEBUG] Success to import application :[public]
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:30:15] [DEBUG] Success to import application :[api]
[2021-09-02 13:30:15] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.web_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.web_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.web_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.web_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.web_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.web_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [api],due to error: No module named 'api.web_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [core],due to error: No module named 'core.web_admin'
[2021-09-02 13:30:15] [ERROR] Fail to import application: [public],due to error: No module named 'public.web_admin'
[2021-09-02 13:30:16] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:30:16] [INFO] The user is Administrator!
[2021-09-02 13:30:16] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 13:30:16] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 13:30:16] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 13:30:16] [DEBUG] The verbose name of this many to many field is: 角色
[2021-09-02 13:30:16] [DEBUG] The verbose name of this many to many field is: 权限组
[2021-09-02 13:30:16] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:30:16] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:30:16] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:30:16] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:30:20] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:30:20] [INFO] The user is Administrator!
[2021-09-02 13:30:23] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:30:23] [INFO] The user is Administrator!
[2021-09-02 13:30:24] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:30:24] [INFO] The user is Administrator!
[2021-09-02 13:30:26] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:30:26] [INFO] The user is Administrator!
[2021-09-02 13:30:26] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 13:30:26] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 13:30:26] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 13:30:26] [DEBUG] The verbose name of this many to many field is: 角色
[2021-09-02 13:30:26] [DEBUG] The verbose name of this many to many field is: 权限组
[2021-09-02 13:30:26] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:30:26] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:30:26] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:30:29] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:30:29] [INFO] The user is Administrator!
[2021-09-02 13:30:46] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:30:46] [INFO] The user is Administrator!
[2021-09-02 13:30:46] [DEBUG] There is a file [<MultiValueDict: {'upload': [<InMemoryUploadedFile: background.jpg (image/jpeg)>]}>] to be uploaded!
[2021-09-02 13:30:46] [DEBUG] The file is: background.jpg
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('api', 'apiproject')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('api', 'templates')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('api', 'testsuit')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('api', 'testcase')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('api', 'scenario')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('api', 'sql')]
[2021-09-02 13:43:17] [DEBUG] Success to import application :[api]
[2021-09-02 13:43:17] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.public_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.public_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.public_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.public_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.public_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.public_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [api],due to error: No module named 'api.public_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [core],due to error: No module named 'core.public_admin'
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('public', 'userprofile')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('public', 'groups')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('public', 'role')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('public', 'firstlayermenu')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('public', 'submenu')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('public', 'businessfunc')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('public', 'retrieval')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('public', 'resource')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('public', 'environment')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('public', 'dbinfo')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('public', 'logserverinfo')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('public', 'configurations')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('public', 'cn_en_map')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('public', 'steplog')]
[2021-09-02 13:43:17] [DEBUG] The application and model name are [('public', 'todolist')]
[2021-09-02 13:43:17] [DEBUG] Success to import application :[public]
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.api_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.api_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.api_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.api_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.api_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.api_admin'
[2021-09-02 13:43:17] [DEBUG] Success to import application :[api]
[2021-09-02 13:43:17] [ERROR] Fail to import application: [core],due to error: No module named 'core.api_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [public],due to error: No module named 'public.api_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.admin],due to error: No module named 'django.contrib.admin.web_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.auth],due to error: No module named 'django.contrib.auth.web_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.contenttypes],due to error: No module named 'django.contrib.contenttypes.web_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.sessions],due to error: No module named 'django.contrib.sessions.web_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.messages],due to error: No module named 'django.contrib.messages.web_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [django.contrib.staticfiles],due to error: No module named 'django.contrib.staticfiles.web_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [api],due to error: No module named 'api.web_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [core],due to error: No module named 'core.web_admin'
[2021-09-02 13:43:17] [ERROR] Fail to import application: [public],due to error: No module named 'public.web_admin'
[2021-09-02 13:43:19] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:43:19] [INFO] The user is Administrator!
[2021-09-02 13:43:19] [DEBUG] There is a file [<MultiValueDict: {'upload': [<InMemoryUploadedFile: background.jpg (image/jpeg)>]}>] to be uploaded!
[2021-09-02 13:43:19] [DEBUG] The file is: background.jpg
[2021-09-02 13:43:21] [ERROR] Fail to upload the file,error as follows: (1062, "Duplicate entry 'ssd' for key 'public_userprofile.user_id'")
[2021-09-02 13:43:29] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:43:29] [INFO] The user is Administrator!
[2021-09-02 13:43:29] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 13:43:29] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 13:43:29] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 13:43:29] [DEBUG] The verbose name of this many to many field is: 角色
[2021-09-02 13:43:29] [DEBUG] The verbose name of this many to many field is: 权限组
[2021-09-02 13:43:29] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:43:29] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:43:29] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:43:29] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:43:36] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:43:36] [INFO] The user is Administrator!
[2021-09-02 13:43:36] [INFO] Prepare to execute operation: [delete_selected]
[2021-09-02 13:43:36] [DEBUG] The selected ids are: [4,7]
[2021-09-02 13:43:37] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:43:37] [INFO] The user is Administrator!
[2021-09-02 13:43:37] [INFO] Prepare to execute operation: [delete_selected]
[2021-09-02 13:43:37] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:43:37] [INFO] The user is Administrator!
[2021-09-02 13:43:37] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 13:43:37] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 13:43:37] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 13:43:37] [DEBUG] The verbose name of this many to many field is: 角色
[2021-09-02 13:43:37] [DEBUG] The verbose name of this many to many field is: 权限组
[2021-09-02 13:43:37] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:43:37] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:43:39] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:43:39] [INFO] The user is Administrator!
[2021-09-02 13:44:00] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:44:00] [INFO] The user is Administrator!
[2021-09-02 13:44:00] [DEBUG] There is a file [<MultiValueDict: {'upload': [<InMemoryUploadedFile: 2.png (image/png)>]}>] to be uploaded!
[2021-09-02 13:44:00] [DEBUG] The file is: 2.png
[2021-09-02 13:44:01] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 13:44:01] [INFO] The user is Administrator!
[2021-09-02 13:44:01] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 13:44:01] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 13:44:01] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 13:44:01] [DEBUG] The verbose name of this many to many field is: 角色
[2021-09-02 13:44:01] [DEBUG] The verbose name of this many to many field is: 权限组
[2021-09-02 13:44:01] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:44:01] [ERROR] Fail to display the field :[reset]
[2021-09-02 13:44:01] [ERROR] Fail to display the field :[reset]
[2021-09-02 16:01:16] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:01:16] [INFO] The user is Administrator!
[2021-09-02 16:01:16] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:01:16] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:01:16] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:01:16] [DEBUG] The verbose name of this many to many field is: 角色
[2021-09-02 16:01:16] [DEBUG] The verbose name of this many to many field is: 权限组
[2021-09-02 16:01:16] [ERROR] Fail to display the field :[reset]
[2021-09-02 16:01:16] [ERROR] Fail to display the field :[reset]
[2021-09-02 16:01:16] [ERROR] Fail to display the field :[reset]
[2021-09-02 16:01:50] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:01:50] [INFO] The user is Administrator!
[2021-09-02 16:01:50] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:01:50] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:01:50] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:01:54] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:01:54] [INFO] The user is Administrator!
[2021-09-02 16:01:54] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:01:54] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:01:54] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:03:20] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:03:20] [INFO] The user is Administrator!
[2021-09-02 16:04:07] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:04:07] [INFO] The user is Administrator!
[2021-09-02 16:04:09] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:04:09] [INFO] The user is Administrator!
[2021-09-02 16:04:09] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:04:09] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:04:09] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:04:40] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:04:40] [INFO] The user is Administrator!
[2021-09-02 16:05:30] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:05:30] [INFO] The user is Administrator!
[2021-09-02 16:05:30] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:05:30] [INFO] The user is Administrator!
[2021-09-02 16:05:30] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:05:30] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:05:30] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:05:33] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:05:33] [INFO] The user is Administrator!
[2021-09-02 16:05:58] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:05:58] [INFO] The user is Administrator!
[2021-09-02 16:05:58] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:05:58] [INFO] The user is Administrator!
[2021-09-02 16:05:59] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:05:59] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:05:59] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:06:02] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:06:02] [INFO] The user is Administrator!
[2021-09-02 16:06:06] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:06:06] [INFO] The user is Administrator!
[2021-09-02 16:06:06] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:06:06] [INFO] The user is Administrator!
[2021-09-02 16:06:06] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:06:06] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:06:06] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:07:14] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:07:14] [INFO] The user is Administrator!
[2021-09-02 16:07:29] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:07:29] [INFO] The user is Administrator!
[2021-09-02 16:07:30] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:07:30] [INFO] The user is Administrator!
[2021-09-02 16:07:30] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:07:30] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:07:30] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:07:33] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:07:33] [INFO] The user is Administrator!
[2021-09-02 16:07:40] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:07:40] [INFO] The user is Administrator!
[2021-09-02 16:07:40] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:07:40] [INFO] The user is Administrator!
[2021-09-02 16:07:40] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:07:40] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:07:40] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:08:18] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:08:18] [INFO] The user is Administrator!
[2021-09-02 16:08:18] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:08:18] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:08:18] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:08:25] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:08:25] [INFO] The user is Administrator!
[2021-09-02 16:08:25] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:08:25] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:08:25] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:08:26] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:08:26] [INFO] The user is Administrator!
[2021-09-02 16:08:40] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:08:40] [INFO] The user is Administrator!
[2021-09-02 16:08:41] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:08:41] [INFO] The user is Administrator!
[2021-09-02 16:08:41] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:08:41] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:08:41] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:08:44] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:08:44] [INFO] The user is Administrator!
[2021-09-02 16:08:56] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:08:56] [INFO] The user is Administrator!
[2021-09-02 16:08:56] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:08:56] [INFO] The user is Administrator!
[2021-09-02 16:08:56] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:08:56] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:08:56] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:09:00] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:09:00] [INFO] The user is Administrator!
[2021-09-02 16:09:00] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:09:00] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:09:00] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:09:07] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:09:07] [INFO] The user is Administrator!
[2021-09-02 16:09:07] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:09:07] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:09:07] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:09:09] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:09:09] [INFO] The user is Administrator!
[2021-09-02 16:09:16] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:09:16] [INFO] The user is Administrator!
[2021-09-02 16:09:17] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 16:09:17] [INFO] The user is Administrator!
[2021-09-02 16:09:17] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 16:09:17] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 16:09:17] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 16:09:19] [DEBUG] 修剪后的全部项目信息:[]
[2021-09-02 16:09:19] [DEBUG] 汇总后的全部项目信息为：[]
[2021-09-02 16:10:41] [DEBUG] 修剪后的全部项目信息:[]
[2021-09-02 16:10:41] [DEBUG] 汇总后的全部项目信息为：[]
[2021-09-02 16:11:08] [DEBUG] 修剪后的全部项目信息:[]
[2021-09-02 16:11:08] [DEBUG] 汇总后的全部项目信息为：[]
[2021-09-02 17:53:34] [DEBUG] 修剪后的全部项目信息:[]
[2021-09-02 17:53:34] [DEBUG] 汇总后的全部项目信息为：[]
[2021-09-02 17:55:39] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 17:55:39] [INFO] The user is Administrator!
[2021-09-02 17:55:39] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 17:55:39] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 17:55:39] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 17:55:46] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 17:55:46] [INFO] The user is Administrator!
[2021-09-02 17:55:46] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 17:55:46] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 17:55:46] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 17:56:15] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 17:56:15] [INFO] The user is Administrator!
[2021-09-02 17:56:15] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 17:56:15] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 17:56:15] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 17:56:15] [DEBUG] The verbose name of this many to many field is: 角色
[2021-09-02 17:56:15] [DEBUG] The verbose name of this many to many field is: 权限组
[2021-09-02 17:56:15] [ERROR] Fail to display the field :[reset]
[2021-09-02 17:56:15] [ERROR] Fail to display the field :[reset]
[2021-09-02 17:56:15] [ERROR] Fail to display the field :[reset]
[2021-09-02 17:56:20] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 17:56:20] [INFO] The user is Administrator!
[2021-09-02 17:56:20] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 17:56:20] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 17:56:20] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 17:56:29] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 17:56:29] [INFO] The user is Administrator!
[2021-09-02 17:56:29] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 17:56:29] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 17:56:29] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 18:02:46] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 18:02:46] [INFO] The user is Administrator!
[2021-09-02 18:02:47] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 18:02:47] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 18:02:47] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 18:02:52] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 18:02:52] [INFO] The user is Administrator!
[2021-09-02 18:02:59] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 18:02:59] [INFO] The user is Administrator!
[2021-09-02 18:03:00] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 18:03:00] [INFO] The user is Administrator!
[2021-09-02 18:03:00] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 18:03:00] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 18:03:00] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 18:03:04] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 18:03:04] [INFO] The user is Administrator!
[2021-09-02 18:03:29] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 18:03:29] [INFO] The user is Administrator!
[2021-09-02 18:03:29] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 18:03:29] [INFO] The user is Administrator!
[2021-09-02 18:03:30] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 18:03:30] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 18:03:30] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 18:04:25] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 18:04:25] [INFO] The user is Administrator!
[2021-09-02 18:04:25] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 18:04:25] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 18:04:25] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 18:04:28] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 18:04:28] [INFO] The user is Administrator!
[2021-09-02 18:05:06] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 18:05:06] [INFO] The user is Administrator!
[2021-09-02 18:05:06] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 18:05:06] [INFO] The user is Administrator!
[2021-09-02 18:05:06] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 18:05:06] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 18:05:06] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 18:05:16] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 18:05:16] [INFO] The user is Administrator!
[2021-09-02 18:05:16] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 18:05:16] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 18:05:16] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 18:05:20] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 18:05:20] [INFO] The user is Administrator!
[2021-09-02 18:05:32] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 18:05:32] [INFO] The user is Administrator!
[2021-09-02 18:05:32] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-02 18:05:32] [INFO] The user is Administrator!
[2021-09-02 18:05:32] [DEBUG] Prepare to execute: delete_selected!
[2021-09-02 18:05:32] [DEBUG] Prepare to execute: valid_selected!
[2021-09-02 18:05:32] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-02 18:05:36] [DEBUG] 修剪后的全部项目信息:[]
[2021-09-02 18:05:36] [DEBUG] 汇总后的全部项目信息为：[]
[2021-09-03 09:37:21] [DEBUG] 修剪后的全部项目信息:[]
[2021-09-03 09:37:21] [DEBUG] 汇总后的全部项目信息为：[]
[2021-09-03 09:37:37] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:37:37] [INFO] The user is Administrator!
[2021-09-03 09:37:37] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:37:37] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:37:37] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:37:45] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:37:45] [INFO] The user is Administrator!
[2021-09-03 09:37:54] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:37:54] [INFO] The user is Administrator!
[2021-09-03 09:37:56] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:37:56] [INFO] The user is Administrator!
[2021-09-03 09:37:56] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:37:56] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:37:56] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:38:04] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:38:04] [INFO] The user is Administrator!
[2021-09-03 09:38:08] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:38:08] [INFO] The user is Administrator!
[2021-09-03 09:38:09] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:38:09] [INFO] The user is Administrator!
[2021-09-03 09:38:09] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:38:09] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:38:09] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:38:15] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:38:15] [INFO] The user is Administrator!
[2021-09-03 09:38:19] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:38:19] [INFO] The user is Administrator!
[2021-09-03 09:38:19] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:38:19] [INFO] The user is Administrator!
[2021-09-03 09:38:19] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:38:19] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:38:19] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:38:31] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:38:31] [INFO] The user is Administrator!
[2021-09-03 09:38:51] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:38:51] [INFO] The user is Administrator!
[2021-09-03 09:38:51] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:38:51] [INFO] The user is Administrator!
[2021-09-03 09:38:51] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:38:51] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:38:51] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:38:57] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:38:57] [INFO] The user is Administrator!
[2021-09-03 09:39:04] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:04] [INFO] The user is Administrator!
[2021-09-03 09:39:04] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:05] [INFO] The user is Administrator!
[2021-09-03 09:39:05] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:39:05] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:39:05] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:39:09] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:09] [INFO] The user is Administrator!
[2021-09-03 09:39:13] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:13] [INFO] The user is Administrator!
[2021-09-03 09:39:13] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:13] [INFO] The user is Administrator!
[2021-09-03 09:39:13] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:39:13] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:39:13] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:39:16] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:16] [INFO] The user is Administrator!
[2021-09-03 09:39:19] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:19] [INFO] The user is Administrator!
[2021-09-03 09:39:19] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:19] [INFO] The user is Administrator!
[2021-09-03 09:39:19] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:39:19] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:39:19] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:39:27] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:27] [INFO] The user is Administrator!
[2021-09-03 09:39:30] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:30] [INFO] The user is Administrator!
[2021-09-03 09:39:31] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:31] [INFO] The user is Administrator!
[2021-09-03 09:39:31] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:39:31] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:39:31] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:39:35] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:35] [INFO] The user is Administrator!
[2021-09-03 09:39:40] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:40] [INFO] The user is Administrator!
[2021-09-03 09:39:40] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:40] [INFO] The user is Administrator!
[2021-09-03 09:39:40] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:39:40] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:39:40] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:39:44] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:44] [INFO] The user is Administrator!
[2021-09-03 09:39:54] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:54] [INFO] The user is Administrator!
[2021-09-03 09:39:54] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:54] [INFO] The user is Administrator!
[2021-09-03 09:39:54] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:39:54] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:39:54] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:39:59] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:39:59] [INFO] The user is Administrator!
[2021-09-03 09:40:06] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:40:06] [INFO] The user is Administrator!
[2021-09-03 09:40:06] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:40:06] [INFO] The user is Administrator!
[2021-09-03 09:40:06] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:40:06] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:40:06] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:40:11] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:40:11] [INFO] The user is Administrator!
[2021-09-03 09:40:20] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:40:20] [INFO] The user is Administrator!
[2021-09-03 09:40:20] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:40:20] [INFO] The user is Administrator!
[2021-09-03 09:40:20] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:40:20] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:40:20] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:40:25] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:40:25] [INFO] The user is Administrator!
[2021-09-03 09:40:25] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:40:25] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:40:25] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:40:27] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:40:27] [INFO] The user is Administrator!
[2021-09-03 09:40:33] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:40:33] [INFO] The user is Administrator!
[2021-09-03 09:40:33] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:40:33] [INFO] The user is Administrator!
[2021-09-03 09:40:33] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:40:33] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:40:33] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:40:39] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:40:39] [INFO] The user is Administrator!
[2021-09-03 09:40:39] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:40:39] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:40:39] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:40:45] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:40:45] [INFO] The user is Administrator!
[2021-09-03 09:40:50] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:40:50] [INFO] The user is Administrator!
[2021-09-03 09:40:50] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:40:50] [INFO] The user is Administrator!
[2021-09-03 09:40:51] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:40:51] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:40:51] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:40:54] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:40:54] [INFO] The user is Administrator!
[2021-09-03 09:40:54] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:40:54] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:40:54] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:41:03] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:41:03] [INFO] The user is Administrator!
[2021-09-03 09:41:11] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:41:11] [INFO] The user is Administrator!
[2021-09-03 09:41:11] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:41:11] [INFO] The user is Administrator!
[2021-09-03 09:41:11] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:41:11] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:41:11] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:41:14] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:41:14] [INFO] The user is Administrator!
[2021-09-03 09:41:14] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:41:14] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:41:14] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:41:17] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:41:17] [INFO] The user is Administrator!
[2021-09-03 09:41:25] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:41:25] [INFO] The user is Administrator!
[2021-09-03 09:41:25] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:41:25] [INFO] The user is Administrator!
[2021-09-03 09:41:25] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:41:25] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:41:25] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:41:29] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:41:29] [INFO] The user is Administrator!
[2021-09-03 09:41:29] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:41:29] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:41:29] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:41:33] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:41:33] [INFO] The user is Administrator!
[2021-09-03 09:41:42] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:41:42] [INFO] The user is Administrator!
[2021-09-03 09:41:42] [DEBUG] Start to match permission: <class 'tuple'>
[2021-09-03 09:41:42] [INFO] The user is Administrator!
[2021-09-03 09:41:42] [DEBUG] Prepare to execute: delete_selected!
[2021-09-03 09:41:42] [DEBUG] Prepare to execute: valid_selected!
[2021-09-03 09:41:42] [DEBUG] Prepare to execute: invalid_selected!
[2021-09-03 09:41:55] [DEBUG] 修剪后的全部项目信息:[]
[2021-09-03 09:41:55] [DEBUG] 汇总后的全部项目信息为：[]
[2021-09-03 09:42:51] [DEBUG] 修剪后的全部项目信息:[]
[2021-09-03 09:42:51] [DEBUG] 汇总后的全部项目信息为：[]
